.github/workflows/quality-gates.yml

```
@@ -45,18 +45,38 @@ jobs:
+            /^LF:/ {LF += $2}
+            /^LH:/ {LH += $2}
+            END {
+              pct = (LF > 0) ? (100.0*LH/LF) : 0;
+              printf "%.2f", pct
+            }' coverage/lcov.info)
+
+          echo "Frontend coverage: ${PCT}% (min ${THRESHOLD}%)"
+          export PCT THRESHOLD
+
+          python3 - <<PY
+import os, sys
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 72: could not find expected ':'

(syntax-check)